### DISCUSSION TITLE
Could we add the `image-rendering: pixelated;` CSS property on the image?

### DISCUSSION THREAD

**Discussion by gablabelle** [willnguyen1312/zoom-image#326] [Ideas]
Hello there!

Saw [this post on X](https://x.com/FrontendMasters/status/1840892656341176348) earlier today and was wondering if we could add the `image-rendering: pixelated;` CSS property on the image so it could improve the quality of the image when scaled.

I'm using the move behaviour and I need to zoom in small images. We can't provide the bigger sizes because we don't want users to potentially download the "good quality" image.

Maybe this could simply be a props since there are multiple options for that CSS property

![Screenshot 2024-10-02 at 15 58 17](https://github.com/user-attachments/assets/35fa2ccb-2fee-4cfc-8935-1980ce4be490)

**Comment by willnguyen1312** [comment]
Hi Gab. Thanks for your input. I think the above should be part of zoomed image's properties specifically [className](https://developer.mozilla.org/en-US/docs/Web/API/Element/className) in this case. I just released a newer version and updated the [docs](https://willnguyen1312.github.io/zoom-image/api/createZoomImageMove.html#type-declaration) for such purpose. Hope it helps :)

**Reply by gablabelle** [reply]
Thanks mate
